invokeai/frontend/web/src/features/modelManagerV2/subpanels/AddModelPanel/StarterModels/StartModelsResultItem.tsx

```diff
@@ -12,17 +12,20 @@ type Props = {
 export const StarterModelsResultItem = ({ result }: Props) => {
   const { t } = useTranslation();
   const allSources = useMemo(() => {
-    const _allSources = [result.source];
+    const _allSources = [{ source: result.source, config: { name: result.name, description: result.description } }];
     if (result.dependencies) {
-      _allSources.push(...result.dependencies.map((d) => d.source));
+      for (const d of result.dependencies) {
+        _allSources.push({ source: d.source, config: { name: d.name, description: d.description } });
+      }
+      // _allSources.push(...result.dependencies.map((d) => {source: d.source, config: {name: d.name, description: d.description}}));
     }
     return _allSources;
   }, [result]);
   const [installModel] = useInstallModel();
 
   const onClick = useCallback(() => {
-    for (const source of allSources) {
-      installModel({ source });
+    for (const s of allSources) {
+      installModel({ source: s.source, config: s.config });
```

Comment:
This is totally fine - no need to change - but wanted to give you a tip for JS object "destructuring" and shorthand for specifying object properties.

```suggestion
    for (const { config, source } of allSources) {
      installModel({ source, config });
```

- You can do `const { propertyName1, propertyName2 } = object` to pull out the named properties as variables.
- When creating an object, if you already have a variable with the same name as the property you'll be assigning the variable to, you can omit the value part of the object definition. So you could do `const object = { propertyName1 }` instead of `const object = { propertyName1: propertyName1 }`.

Reply:
Thanks for the guidance. I'd tried various versions of that shorthand, but hadn't got the position of the `const` right.